modules/meta.analysis/R/run.meta.analysis.R

```diff
@@ -190,7 +190,7 @@ run.meta.analysis <- function(pfts, iterations, random = TRUE, threshold = 1.2,
   dbcon <- PEcAn.DB::db.open(database)
   on.exit(PEcAn.DB::db.close(dbcon), add = TRUE)
 
-  result <- lapply(pfts, run.meta.analysis.pft, iterations = iterations, random = random, 
+  result <- lapply(pfts, run.meta.analysis.pft(update = settings$meta.analysis$update), iterations = iterations, random = random, 
```

Comment:
This isn't how lapply works. You need to pass update as one of the arguments to lapply, which then passes them to the function for each pft, same as is being done for the other arguments here.